internal/controllers/lvmcluster/controller.go

```diff
@@ -210,6 +208,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 func (r *Reconciler) reconcile(ctx context.Context, instance *lvmv1alpha1.LVMCluster) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
 
+	instance.Status.Conditions = computeInitialConditions()
```

Comment:
I suggest to change the condition type to https://github.com/openshift/lvm-operator/blob/ab9df4c80d2ac20eb9818d3ff44a337f2f8918ca/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1497 and then make use of these conditions

Then you can also use functions like https://github.com/openshift/lvm-operator/blob/0d154af54cbd6eef68fb5ee421ed4a37f9b8e017/vendor/k8s.io/apimachinery/pkg/api/meta/conditions.go#L31

WDYT?